app/views/layouts/mailer.html.erb

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" content="width=device-width, initial-scale=1.0">
     <title></title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
-    <%= stylesheet_link_tag "/css/mailer.css" %>
+    <%= stylesheet_link_tag :mailer %>
```

Comment:
External CSS won't work in most of email clients including Gmail

Reply:
oh okk will fix !

Reply:
@tachyons can we use `premailer-rails` gem for injecting the CSS ?

Reply:
@tachyons added mailer.css to asset precompile
and premier already initialized, so the css's already converted to inline CSS before sending

By inspecting the source details of mail from MailCatcher we can see the stylesheet link converted to inline CSS by premailer
![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/459ae8a8-6652-4daa-b196-8db6a2744f18)